package.json

```diff
@@ -53,14 +53,15 @@
     "types-ramda": "0.29.4"
   },
   "dependencies": {
-    "@chakra-ui/focus-lock": "2.1.0",
-    "@chakra-ui/react": "2.8.1",
-    "@chakra-ui/system": "2.6.2",
+    "@chakra-ui/focus-lock": ">=2.1.0",
+    "@chakra-ui/react": ">=2.8.1",
```

Comment:
Did you make all the ">=" updates? Did you encounter any issues while installing packages?

Reply:
Oops yes accidentally, I had something up with esbuild, they're locked now